src/expr/parsing.hpp

```diff
@@ -15,7 +15,7 @@
 #include <cstdint>
 #include <string>
```

Comment:
Are `parsing.hpp` and `parsing.cpp` in evaluator and not in parser only because of exceptions?

Reply:
These are helper functions and I think belong separately from `mg-parser`.  Am I missing something?

Reply:
Nope I thought about moving `parser.cpp` and `parse.hpp` to `mg-parse`, hence the name, but yeah now I see it is used in `mg-expr`